WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailCommentsHeader.swift

```diff
@@ -4,12 +4,44 @@ class ReaderDetailCommentsHeader: UITableViewHeaderFooterView, NibReusable {
 
     static let estimatedHeight: CGFloat = 80
     @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet private weak var followButton: UIButton!
+    private let buttonTitle = NSLocalizedString("Follow Conversation", comment: "Button title. Follow the comments on a post.")
 
     override func awakeFromNib() {
         super.awakeFromNib()
+        configureView()
+    }
+
+}
+
+private extension ReaderDetailCommentsHeader {
+
+    func configureView() {
         contentView.backgroundColor = .basicBackground
+        addBottomBorder(withColor: .divider)
+        configureTitle()
+        configureButton()
+    }
+
+    func configureTitle() {
         titleLabel.textColor = .text
         titleLabel.font = WPStyleGuide.serifFontForTextStyle(.title3, fontWeight: .semibold)
     }
 
+    func configureButton() {
+        guard FeatureFlag.followConversationPostDetails.enabled else {
```

Comment:
I think we also need to check if follow conversation is enabled for the particular post. This can be checked via `FollowCommentsService`: 

https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Services/FollowCommentsService.swift#L33-L37

Or if the service is not available in this file, you can directly check the post's `canSubscribeComments` boolean property.

Reply:
Oh sorry! I failed to note that I intentionally didn't add any button toggling logic in this PR. Thanks for the tip though, it'll come in handy later when I do!